backend/drm/drm.c

```diff
@@ -1304,8 +1304,12 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 				wl_list_insert(&wlr_conn->output.modes, &mode->wlr_mode.link);
 			}
 
+			bool is_mst = false;
+			if (strstr(wlr_conn->props.path, "mst:"))
```

Comment:
`wlr_conn->props.path` is an `uint32_t`, need to `drmModePropertyGetBlob` it :P

Reply:
Damn! I thought I could get away cheap :D